models/src/main/scala/coop/rchain/models/Pretty.scala

```diff
@@ -77,10 +77,7 @@ trait PrettyInstances extends PrettyDerivation {
 
   implicit def prettyCoeval[A: Pretty]: Pretty[Coeval[A]] =
     (value: Coeval[A], indentLevel: Int) =>
-      value match {
-        case Now(a: A)    => s"Coeval.Now(${Pretty[A].pretty(a, indentLevel)})"
-        case _: Coeval[_] => s"Coeval(???)"
-      }
+      s"Coeval.now(${Pretty[A].pretty(value.value, indentLevel)}) /* was Coeval.${value.getClass.getSimpleName} */"
```

Comment:
We don't always use `Coeval.now` but for the tests it should be OK.

Reply:
Yes, the original constructor used is pretty-printed in a comment :)

Reply:
I meant that we use `Coeval.delay` in some places and now it will be `Coeval.now`. But as I said, for tests it doesn't matter much.